typescript/cli/src/commands/options.ts

```diff
@@ -147,3 +147,19 @@ export const dryRunOption: Options = {
   default: false,
   alias: ['d', 'dr'],
 };
+
+export const chainCommandOption: Options = {
+  type: 'string',
+  description: 'The specific chain to perform operations with.',
+};
+
+export const addressCommandOption: Options = {
+  type: 'string',
+  description: 'An onchain address.',
+};
```

Comment:
The description's maybe too vague here. Maybe this could take an optional `description` param?

Reply:
good call, I originally had more verbose descriptions initially but dumbed down to de-dupe